sdk/data-generators/number-stream-generator/src/main.rs

```diff
@@ -104,7 +104,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     let encode = pinecone::to_vec(&dataset2)?;
 
     let mut file = File::create(format!("{}-2.txt", file_prefix))?;
-    file.write_all(format!("{:?}\n{:?}", dataset2_txt, encode).as_bytes())?;
+    file.write_all(format!("{:?}\n{:?}", dataset2_txt, dataset2).as_bytes())?;
```

Comment:
Why this change?

Reply:
This is a minor bug fix. It prints out a human-readable copy of the generated dataset. Previously, it printed out the binary encoding.